docs/UserGuide.md

```diff
@@ -85,30 +85,20 @@ Shows a message explaining how to access the help page.
 
 Format: `help`
 
-#### Exiting the program : `exit`
-
-Exits the program.
-
-Format: `exit`
-
-#### Undo/redo a command: `undo`/`redo`
-
-Undoes the most recent command that changed the contact list or task list.
+#### Listing all persons : `list`
 
-Format: `undo`/`redo`
-* `undo`/`redo` will not work on commands that does not change the contact list or task list, eg. mail.
+Shows a list of all persons in the student roster.
 
-Examples:
-* `delete 1` will delete the first person on the list. `undo` will then revert the contact list to contain the person deleted.
+Format: `list`
 
-### Contact List/Student Roster commands
```

Comment:
Any reason why you chose to leave this out? This was part of our comments we obtained from CS2101's feedback. Also, if we leave this out, the nearest"back to top" will now be far away from the rest of the commands under this block.